react-native-aztec/src/AztecView.js

```diff
@@ -91,59 +87,65 @@ class AztecView extends React.Component {
   }
 
   _onFocus = (event) => {
-    if (!this.props.onFocus) {
-      return;
-    }
-
+    this.focused = true;
     const { onFocus } = this.props;
-    onFocus(event);
+
+    if (onFocus) {
+      onFocus(event);
+    }
   }
   
   _onBlur = (event) => {
+    this.focused = false;
     this.selectionEndCaretY = null;
-    TextInputState.blurTextInput(ReactNative.findNodeHandle(this));
+    const { onBlur } = this.props;
 
-    if (!this.props.onBlur) {
-      return;
+    if (onBlur) {
+      onBlur(event);
     }
-
-    const { onBlur } = this.props;
-    onBlur(event);
   }
 
   _onSelectionChange = (event) => {
-    if ( this.props.onSelectionChange ) {
+    if (this.focused && this.props.onSelectionChange) {
```

Comment:
Aren't we able to do the same thing with the old function?

isFocused = () => {		 
     const focusedField = TextInputState.currentlyFocusedField();		
     return focusedField && ( focusedField === ReactNative.findNodeHandle(this) );		
   }